src/execution.jl

```diff
@@ -250,7 +250,11 @@ end
                 close(io)
                 # atomic move
                 mkpath(dirname(path))
-                Base.rename(tmppath, path, force=true)
+                @static if VERSION >= v"1.12.0-DEV.1023"
+                    mv(tmppath, path; force=true)
```

Comment:
Should 
```julia
tmppath, io = mktemp(;cleanup=false)
```
 on line 247 be replaced with:
```julia
tmppath, io = mktemp(dirname(path); cleanup=false)
```
to ensure the temp file is in the same directory as path, like is done in "rtlib.jl" line 153?

Reply:
Probably yes, because otherwise we can't guarantee the operation actually can be executed atomically.